### DISCUSSION TITLE
Sample Sign in and Sign up pages

### DISCUSSION THREAD

**Discussion by ibalat** [livewire/flux#920] [Feature Requests] [answered]
Hi, please you can add sample sign in and sign up pages to document. I want to refer them to generate auth pages.

**Comment by jeffchown** [comment, marked as answer]
@ibalat Have you looked at Laravel's Breeze (https://laravel.com/docs/11.x/starter-kits#breeze-and-livewire) and/or Jetstream (https://jetstream.laravel.com/introduction.html).

You can use their login/auth pages as starting points then 'Fluxify' them.
